### PR TITLE
fix(RELEASE-1061): linting issues in collect-gh-params task

### PR DESCRIPTION
**Pull request by mmalina**

### CODE REVIEW

File: .github/workflows/lint.yaml
Comment (mmalina) [inline .github/workflows/lint.yaml]: Remove changes before merging.

**Review by johnbieren — COMMENTED** [review]
LGTM, will approve once workflow change is removed

File: .github/workflows/lint.yaml
Comment (mmalina) [reply .github/workflows/lint.yaml]: Removed. Ready to be merged.

**Comment by mmalina** [comment]
> LGTM, will approve once workflow change is removed

OK, thanks. Please check now.

**Review by johnbieren — APPROVED** [review]

**Review by happybhati — APPROVED** [review]

**Pull request merged**
